### PR TITLE
Change file operations to use 'with'

### PR DESCRIPTION
**Pull request by tntexplosivesltd**
Python best practice when opening files is to use the 'with' statement
to make sure they're closed/flushed.
See https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files

Also changes a comment to be more in line with what it's doing

### CODE REVIEW

**Comment by CorruptComputer** [comment]
There seems to be an issue with opening the VDF:

image


Is there a new requirement to install with these changes?

File: ProtonDB-to-Steam-Library.py [hunk 1]
```diff
@@ -203,7 +209,8 @@ def main(args):
         sharedconfig_path = find_sharedconfig()
 
     print("Selected: {}".format(sharedconfig_path))
-    sharedconfig = vdf.load(open(sharedconfig_path))
+    with open(args.sharedconfig_path) as sharedconfig_vdf:
```
Comment (thomas-phillips-nz) [inline ProtonDB-to-Steam-Library.py]: Ah, I see my mistake. Copy-paste error from above. This should read:
` with open(sharedconfig_path) as sharedconfig_vdf`

Comment (tntexplosivesltd) [reply ProtonDB-to-Steam-Library.py]: Oops different account. Pushed a new commit.

**Comment by tntexplosivesltd** [comment]
This is ready to be reviewed now.

**Comment by CorruptComputer** [comment]
Looks good to me :+1:

**Pull request merged**